### ISSUE TITLE
Stop calculating the width on every resize tick

### ISSUE THREAD

**Issue by alexnewmannn** [tiberiualex/Greedy-Pure#3]
We may be able to cache some elements widths, would need some investigation but we would also benefit from adding a debounce to the resize event

**Comment by tiberiualex** [comment]
Yup, the plan is to pre-calculate the widths at the beginning. Also, I'll add the debounce once I find an elegant solution for it :+1:

**Comment by tiberiualex** [comment]
The widths are now pre-calculated. I'm still considering if it's worth adding the throttle.
